src/plantumlv2/pu_manager.py

```diff
@@ -213,7 +201,8 @@ def _filter_packages(
                     filtered_packages_set.add(package)
 
             if isinstance(package_view, dict):
-                filter_path = package_view["packagePath"].replace(".", "/")
```

Comment:
there are a bit too many `isinstance` checks in this codebase for my taste :)

Reply:
haha, now I saw that it's not only my taste, but also flake8 that @mathiasbk1 integrated complains about the isinstance: 

![image](https://github.com/archlens/ArchLens/assets/464519/1355484d-4a07-4336-89d4-68c483d27ef8)

is there something we can do about this?

Reply:
We can probably attempt to remove some of these "isInstance" uses, however, should this maybe be another PR.

Reply:
ok. let's do that!